api/service/stagedstreamsync/stage_lastmile.go

```diff
@@ -25,9 +25,9 @@ func NewStageLastMile(cfg StageLastMileCfg) *StageLastMile {
 	}
 }
 
-func NewStageLastMileCfg(ctx context.Context, bc core.BlockChain, db kv.RwDB) StageLastMileCfg {
+func NewStageLastMileCfg(bc core.BlockChain, db kv.RwDB) StageLastMileCfg {
 	return StageLastMileCfg{
-		ctx: ctx,
+		ctx: context.TODO(),
```

Comment:
Touching context like this could possibly make same issues as we had for disk cache. Please take a look at pr #4450

Reply:
```ctx``` is ```context.Background()```. No real changes here.

Reply:
if no real changes, then why we should even do it? It would be good if you share how you test this part.